apps/mobile/package.json

```diff
@@ -91,6 +91,7 @@
     "expo-router": "3.5.14",
     "expo-secure-store": "13.0.2",
     "expo-splash-screen": "0.27.4",
+    "expo-squircle-view": "1.1.0",
```

Comment:
Do we need this in the mobile app + the UI package or can we just have it in the UI package and consume that component?

Reply:
Good point, looking at other native packages in the UI this might be unnecessary.